sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/CleanupHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Iot.Hub.Service.Models;
+
+namespace Azure.Iot.Hub.Service.Samples
+{
+    /// <summary>
+    /// Helper class to help cleanup resources created by the sample runs.
+    /// </summary>
+    public static class CleanupHelper
+    {
+        public async static Task DeleteAllDevicesInHubAsync(IotHubServiceClient hubClient)
+        {
+            Console.WriteLine($"\nTrying to clean up all devices\n");
+
+            try
+            {
+                AsyncPageable<TwinData> asyncPageableResponse = hubClient.Devices.GetTwinsAsync();
```

Comment:
This is where I felt having a list on Device Identities might have been helpful.  Maybe something to revisit. If not the overloaded method, we can have some custom implementation to get all device identities. If there is a one-liner to do this already and i'm missing it, let me know.